diff --git a/node_modules/@reach/menu-button/es/index.js b/node_modules/@reach/menu-button/es/index.js
index 69c1694..9a60a30 100644
--- a/node_modules/@reach/menu-button/es/index.js
+++ b/node_modules/@reach/menu-button/es/index.js
@@ -267,6 +267,7 @@ var MenuListImpl = React.forwardRef(function (_ref10, _ref11) {
       },
       onKeyDown: wrapEvent(onKeyDown, function (event) {
         if (event.key === "Escape") {
+          event.stopPropagation();
           setState(close);
         } else if (event.key === "ArrowDown") {
           event.preventDefault(); // prevent window scroll
